internal/cluster/clusters.go

```diff
@@ -42,7 +42,7 @@ func (c *Clusters) Exists(organizationID uint, name string) (bool, error) {
 		return false, errors.Wrap(err, "could not check cluster existence")
 	}
 
-	return existingCluster.ID == 0, nil
+	return existingCluster.ID != 0, nil
```

Comment:
`existingCluster.ID > 0` I think it's safer

Reply:
to be honest, I don't really know the intents of this check, but would think that it's checked against the default value of `uint` here.